### PR TITLE
Add !define plugin

### PR DESCRIPTION
**Pull request by TetraEtc**
This PR adds in a new plugin - `!define` 

This utilizes the Oxford dictionary API.

It requires two new environment variables
- `OXFORD_APP_ID`
- `OXFORD_APP_KEY`
-  (OPTIONAL) `OXFORD_LANG_CODE` - For languages other than English.

API Credentials can be registered for here: https://developer.oxforddictionaries.com/

Tests for this is a little difficult without putting API keys... somewhere.

### CODE REVIEW

**Comment by TetraEtc** [comment]
@llimllib Is there anything you're wanting me to do with this at all?

**Comment by llimllib** [comment]
Sorry, I think I wrote the review as soon as you put out this PR and never remembered to save it!

**Comment by TetraEtc** [comment]
All good. I'll attempt to get this done soon! 

No idea why those print statements are still in there... must have left them in from debugging!

File: limbo/plugins/oxford.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
+"""!define - Returns the oxford dictionary definition of the given term"""
+import os
+import re
+import requests
+
+def define(word):
+    app_id = os.environ.get('OXFORD_APP_ID', '')
```
Comment (TetraEtc) [inline limbo/plugins/oxford.py]: Can you point to an example of how you want this in particular done? I'm looking at the weather plugin as one of the few other plugins that have an API key, and they're in in a similar way

Comment (llimllib) [inline limbo/plugins/oxford.py]: hah yeah it does, and it's also not a big deal. Dunno what I was thinking about when I wrote that.

**Comment by TetraEtc** [comment]
This should be good to merge in then :)

**Pull request merged**